src/Microsoft.ML.Transforms/Text/TextNormalizing.cs

```diff
@@ -218,6 +218,11 @@ public Mapper(TextNormalizingTransformer parent, DataViewSchema inputSchema)
             public void SaveAsOnnx(OnnxContext ctx)
             {
                 Host.CheckValue(ctx, nameof(ctx));
+
+                const int minimumOpSetVersion = 11;
```

Comment:
>11 [](start = 48, length = 2)

I think this should be version 10. According to the ONNX operators documentation, `StringNormalizer` was added in version 10, and `Squeeze` and `Unsqueeze` were updated in version 11, but they existed before that as well (the update in version 11 was to support passing negative numbers in the `axes` attribute, which we are not using here).

Reply:
Thanks for mentioning the ONNX operator documentation. I was not aware of that before. I'll take a look and fix those related.